app/src/main/java/com/xmartlabs/template/ui/StartActivity.java

```diff
@@ -2,17 +2,21 @@
 
 import android.content.Intent;
 import android.os.Bundle;
+import android.support.annotation.NonNull;
 
 import com.xmartlabs.template.controller.AuthController;
 import com.xmartlabs.template.controller.SessionController;
 import com.xmartlabs.template.model.Session;
+import com.xmartlabs.template.ui.mvp.BaseMvpActivity;
+import com.xmartlabs.template.ui.mvp.BaseMvpPresenter;
+import com.xmartlabs.template.ui.mvp.MvpView;
 
 import javax.inject.Inject;
 
 /**
  * Created by santiago on 31/08/15.
  */
-public class StartActivity extends BaseActivity {
+public class StartActivity extends BaseMvpActivity<MvpView, BaseMvpPresenter<MvpView>> {
```

Comment:
Start activity hasn't got a defined view, right?. In my opinion it shouldn't extend a `BaseMvpActivity`